### PR TITLE
Fix foreach-json.bal

### PR DESCRIPTION
**Pull request by nuwandiw**

### CODE REVIEW

File: tests/ballerina-test/src/test/resources/test-src/statements/foreach/foreach-json.bal [hunk 1]
```diff
@@ -77,14 +81,18 @@ struct Protocol {
     string url;
 }
 
-function testJSONToStructCast () returns (string) {
+function testJSONToStructCast () returns (string) | error {
```
Comment (afkham) [inline tests/ballerina-test/src/test/resources/test-src/statements/foreach/foreach-json.bal]: Better to change this to 
```ballerina
returns string|error
```

File: tests/ballerina-test/src/test/resources/test-src/statements/foreach/foreach-json.bal [hunk 2]
```diff
@@ -26,9 +26,13 @@ function testJSONArray () returns (string) {
     return output;
 }
 
-function testArrayOfJSON () returns (string) {
+function testArrayOfJSON () returns (string) | error {
```
Comment (afkham) [inline tests/ballerina-test/src/test/resources/test-src/statements/foreach/foreach-json.bal]: Better to change this to
```ballerina
returns string|error
```

File: tests/ballerina-test/src/test/java/org/ballerinalang/test/statements/foreach/ForeachJSONTests.java [hunk 3]
```diff
@@ -97,7 +97,7 @@ public void testJSONNull() {
         Assert.assertEquals(returns[0].stringValue(), result);
     }
 
-    @Test
+    @Test (enabled = false)
```
Comment (afkham) [inline tests/ballerina-test/src/test/java/org/ballerinalang/test/statements/foreach/ForeachJSONTests.java]: Why is this test disabled?

File: tests/ballerina-test/src/test/resources/test-src/statements/foreach/foreach-json.bal (hunk 2 above)
Comment (nuwandiw) [reply tests/ballerina-test/src/test/resources/test-src/statements/foreach/foreach-json.bal]: Done

File: tests/ballerina-test/src/test/java/org/ballerinalang/test/statements/foreach/ForeachJSONTests.java (hunk 3 above)
Comment (nuwandiw) [reply tests/ballerina-test/src/test/java/org/ballerinalang/test/statements/foreach/ForeachJSONTests.java]: json to strut casting was not working by the time of earlier commit. Now it's working fine. Enabled the test again.

**Pull request merged**